src/m365/spo/commands/contenttype/contenttype-add.ts

```diff
@@ -75,8 +77,31 @@ class SpoContentTypeAddCommand extends SpoCommand {
           cb(new CommandError(response.ErrorInfo.ErrorMessage));
           return;
         }
-        
-        cb();
+
+        const options: SpoContentTypeGetCommandOptions = {
+          webUrl: args.options.webUrl,
+          listTitle: args.options.listTitle,
+          id: args.options.id,
+          output: args.options.output ?? 'json',
+          debug: this.debug,
+          verbose: this.verbose
+        };
+        Cli.executeCommandWithOutput(SpoContentTypeGetCommand as Command, { options: { ...options, _: [] } })
+          .then((res: CommandOutput): void => {
+            if (this.debug) {
+              logger.logToStderr(res.stderr);
+            }
+            let result = res.stdout;
+            if (options.output === Output[Output.json]) {
+              result = JSON.parse(result);
+            }
+
+            logger.log(result);
```

Comment:
By always returning JSON in the previous call we can simplify the logger output here.

```suggestion
            logger.log(JSON.parse(res.stdout));
```

When using `--output text` the output is automatically formatted by the CLI formatter.

Reply:
done